tools/docker/riak/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM hectcastro/riak
```

Comment:
Can we add a specific version here? If the version behind this tag changes and our image has to be rebuilt after the fact, we might run into inconsistencies.